angular/package.json

```diff
@@ -0,0 +1,176 @@
+{
+  "name": "angular2-webpack-starter",
+  "version": "5.4.1",
+  "description": "An Angular 2 Webpack Starter kit featuring Angular 2 (Router, Http, Forms, Services, Tests, E2E, Coverage), Karma, Protractor, Jasmine, Istanbul, TypeScript, and Webpack by AngularClass",
+  "keywords": [
+    "angular2",
+    "webpack",
+    "typescript"
+  ],
+  "author": "Patrick Stapleton <patrick@angularclass.com>",
+  "homepage": "https://github.com/angularclass/angular2-webpack-starter",
+  "license": "MIT",
+  "scripts": {
```

Comment:
Doel je op de `scripts`? Deze zou ik zeker laten staan namelijk, hier staan veel verschillende nuttige commando's in voorgeconfigureerd die je vroeg of laat wel gaat gebruiken.